test/cppsuite/test_harness/workload_generator.h

```diff
@@ -243,6 +256,162 @@ class workload_generator : public component {
         return (cursor->update(cursor));
     }
 
+    bool
+    validate()
+    {
+        WT_CURSOR *cursor;
+        WT_SESSION *session;
+
+        std::string collection_name;
+        /* Collections which have not been dropped. */
+        std::vector<std::string> active_collections;
+
+        int key;
+        const char *key_collection_name;
```

Comment:
Please one line all these declarations and move them to the top. (without whitespace). e.g.
```
int key, key_timestamp;
const char *key_collection_name, *value;
```